__tests__/app.test.js

```diff
@@ -268,3 +268,26 @@ describe("get/api/users", () => {
     return request(app).get("/api/this-is-not-a-route").expect(404);
   });
 });
+
+describe("get/api/articles(topic query)", () => {
+  test("GET:200 should respond with articles with topic of cats", () => {
+    return request(app)
+      .get("/api/articles")
+      .query({ topic: "cats" })
+      .expect(200)
+      .then(({ body }) => {
+        expect(
+          body.forEach((article) => {
```

Comment:
whenever we do a forEach to test the array that comes back we want to make sure it's not a false positive given to us by a empty array. so please make sure the array is also not empty